build.gradle

```diff
@@ -80,6 +80,8 @@ dependencies {
 	api 'org.quiltmc:quilt-json5:1.0.0'
 	include 'org.quiltmc:quilt-loader-sat4j:2.3.5.1'
 
+	api 'com.electronwill.night-config:core:3.6.5'
+	api 'com.electronwill.night-config:toml:3.6.5'
```

Comment:
Like i5 said: we need this added to the `src/main/resources/quilt_installer.json` and `src/main/resources/fabric-installer.launchwrapper.json` files - I'm not completely sure where both are used, but it should be added to both to be safe.

Reply:
What are your and @TheGlitch76's thoughts on shadowing and relocating NightConfig rather than adding it to the dependency files? Since it is purely impl and not a library we control, this would allow us to avoid mirroring it on our Maven server.

Reply:
I'd be okay with that too!